test/Nancy.Tests/NoAppStartupsFixture.cs

```diff
@@ -44,9 +44,13 @@ public IEnumerable<InstanceRegistration> InstanceRegistrations
 
         private static void ThrowWhenNoAppStartupsFixtureRuns()
         {
+#if CORE
+            if ( Environment.StackTrace.Contains("Nancy.Tests.NoAppStartupsFixture"))

```

Comment:
less robust than `StackTrace().GetFrames()`.
Although if you're not worried about it, it might be cleaner to collapse to this.